src/integrationtests/java/com/aws/greengrass/integrationtests/status/EventFleetStatusServiceTest.java

```diff
@@ -233,8 +233,100 @@ void GIVEN_jobs_deployment_WHEN_deployment_finishes_THEN_status_is_uploaded_to_c
     }
 
     @Test
-    void WHEN_deployment_bumps_up_component_version_THEN_status_of_new_version_is_updated_to_cloud() throws Exception {
-        ((Map) kernel.getContext().getvIfExists(Kernel.SERVICE_TYPE_TO_CLASS_MAP_KEY).get()).put("plugin",
+    void GIVEN_local_deployment_WHEN_deployment_finishes_THEN_status_is_uploaded_to_cloud() throws Exception {
+        CountDownLatch fssPublishLatch = new CountDownLatch(1);
+        logListener = eslm -> {
+            if (eslm.getEventType() != null
+                    && eslm.getEventType().equals("fss-status-update-published")
+                    && eslm.getMessage().contains("Status update published to FSS")) {
+                fssPublishLatch.countDown();
+            }
+        };
+        Slf4jLogAdapter.addGlobalListener(logListener);
```

Comment:
wrap this in a try/catch so the logListener gets removed on error

You can do:
```
 try (AutoCloseable ignored = TestUtils.createCloseableLogListener(listener)) {
   // test
 }
 ```

Reply:
Good call. Changed for the new tests and the pre-existing ones.